resources/assets/less/bem/page-extra.less

```diff
@@ -35,6 +35,11 @@
     padding: 0;
   }
 
+  &--me-section {
+    padding-right: 0px;
```

Comment:
hmmm

![](https://s.myconan.net/2018-03/firefox_2018-03-28_20-26-53.png)

Reply:
I think most of scrollbars like does follow the padding (as in doesn't stick to the edge).